### PR TITLE
CORE-8024 http-rpc-security-read -> rest-security-read

### PR DESCRIPTION
**Pull request by pmck91**

### CODE REVIEW

File: libs/http-rpc/http-rpc-stubs-impl/src/main/kotlin/net/corda/httprpc/endpoints/impl/HelloRestResourceImpl.kt [hunk 1]
```diff
@@ -1,41 +0,0 @@
...
-interface HelloRestResource : RestResource {
-
-    @HttpPOST(description = "This method produces a greeting phrase for the addressee.",
-        responseDescription = "A greeting phrase for the addressee")
-    fun greet(@RestQueryParameter(description = "An arbitrary name can be used for the greeting.") addressee: String): String
-}
-
-@Component(service = [PluggableRestResource::class])
-@Suppress("unused")
-class HelloRestResourceImpl : HelloRestResource, PluggableRestResource<HelloRestResource>, RestResource {
```
Comment (vkolomeyko) [inline libs/http-rpc/http-rpc-stubs-impl/src/main/kotlin/net/corda/httprpc/endpoints/impl/HelloRestResourceImpl.kt]: This files appears to be deleted. Was it accidental or intentional?

Comment (pmck91) [reply libs/http-rpc/http-rpc-stubs-impl/src/main/kotlin/net/corda/httprpc/endpoints/impl/HelloRestResourceImpl.kt]: This is not on purpose. PR 2979 is responsible for that file, I will merge it into this once it  gets a green build

**Review by ben-millar — DISMISSED** [review]
LGTM pending 🟢 build

**Review by ronanbrowne — APPROVED** [review]
LGTM (build)

**Review by ben-millar — APPROVED** [review]

**Pull request merged**